CMakeLists.txt

```diff
@@ -172,7 +172,7 @@ if(${MODULESPACK} MATCHES "ON" AND NOT WIN32)
 endif()
 
 if(NOT WIN32)
-	if(${PROTOCOL_ARPING} MATCHES "ON" AND NOT ${CMAKE_SYSTEM_PROCESSOR} MATCHES "^aarch64")
```

Comment:
What the use of this change?

Reply:
on Raspberry pi, aarch64, cmake wouldn't autodetect libpcap and cause build failure.

[ 93%] Linking C executable pilight-daemon
    /usr/bin/ld: libpilight.so: undefined reference to `pcap_sendpacket'
    /usr/bin/ld: libpilight.so: undefined reference to `pcap_dispatch'
    /usr/bin/ld: libpilight.so: undefined reference to `pcap_close'
    /usr/bin/ld: libpilight.so: undefined reference to `pcap_open_live'
    /usr/bin/ld: libpilight.so: undefined reference to `pcap_geterr'
    /usr/bin/ld: libpilight.so: undefined reference to `pcap_get_selectable_fd'
    /usr/bin/ld: libpilight.so: undefined reference to `pcap_setnonblock'
    collect2: error: ld returned 1 exit status
    make[2]: *** [CMakeFiles/pilight-daemon.dir/build.make:101: pilight-daemon] Error 1
    make[1]: *** [CMakeFiles/Makefile2:669: CMakeFiles/pilight-daemon.dir/all] Error 2
    make: *** [Makefile:156: all] Error 2
    
   sorry I forgot to add exact error message in the commit. I just pushed a new version.